common/content/finder.js

```diff
@@ -124,15 +128,27 @@ var Finder = Module("finder", {
 
             // PDF.JS files are different, they use their own messages to communicate the results.
             // So we piggyback the end changes to the findbar when there are any.
-            findbar._original_updateControlState = findbar.updateControlState;
+            findbar._vimpbackup_updateControlState = findbar.updateControlState;
```

Comment:
Yeah, it was just a thought that occurred to me. I have seen add-ons backup ```.method``` into the same object as ```._method```, and I've also seen variants of ```.original_method``` and ```.backup_method``` as well. I thought specificity to Vimperator wouldn't hurt.